docs/pages/installation.mdx

```diff
@@ -14,25 +18,32 @@ If you are new to Teleport, we recommend following our [getting started guides](
 ## Linux
 
 The following examples install the 64-bit version of Teleport binaries, but
-32-bit (i386) and ARM binaries are also available. Check the [Latest
-Release](https://goteleport.com/download/) page for the most
-up-to-date information.
+32-bit (i386) and ARM binaries are also available. Check the
+[Downloads](https://goteleport.com/download/) page for the most up-to-date
+information.
 
 (!docs/pages/includes/permission-warning.mdx!)
 
 (!docs/pages/includes/install-linux.mdx!)
 
 ## Docker
 
-Please follow our [Getting started with Teleport using Docker](./setup/guides/docker.mdx) or with [Teleport Enterprise using Docker](enterprise/getting-started.mdx#run-teleport-enterprise-using-docker) for install and setup instructions.
+Please follow our
```

Comment:
```suggestion
Please follow
```

Since "follow" modifies a proper name, I think we can remove the possessive. We could also leave the "our" in and add "guide" after the names of the guides.